test/NovaVaultV2.t.sol

```diff
@@ -288,7 +288,12 @@ contract NovaVaultV2Test is Test {
         assertEq(underlyingAmount, IERC20(underlyingAddress).balanceOf(alice));
     }
 
-    function testNovaVaultV2DepositShouldFail()
+    function testTransferOwnership() public {
```

Comment:
can you also check that alice can / cannot do some actions (like allow a contract) when she is / is not owner. it would enable testing the modifier

Reply:
my bad, it's done below

Reply:
the function for that is `testShouldFailBecauseCallerIsNotOwner()`